### ISSUE TITLE
Idea

### ISSUE THREAD

**Issue by Lecrapouille** [marmust/internet-scanner#1]
Hi ! Nice YT video, here is a funny idea you can add in your project like I've done with this POC project https://github.com/Lecrapouille/IslandedBrowser you can display the graph of your browser bookmarks (ie graph links are keyword or folders). And when you click on one of parent node this will call Firefox opening all sub children at once.

**Comment by marmust** [comment]
just checked out the url super cool idea especially the integration with the already existing Firefox browser!
i also had an idea to change the unity project to render graphs saved as txt files (or something similar) so it could render websites, local directories, bookmarks as you mentioned, etc...
that could also let you save and load graphs
but well see if i get to it

**Comment by Lecrapouille** [comment]
Firefox JSON bookmarks is kind of "graphs saved as txt" ;) With visitor pattern traversing your graph structure and generate any kind of file format, networkx is straightforward for this

**Comment by marmust** [comment]
yep turns out these graph rendering things are more common than i thought :)
